### PR TITLE
doc: Add script to generate sphinx doc 

### PR DESCRIPTION
**Pull request by ki4070ma**
For https://github.com/appium/python-client/issues/439

### Published doc(temporal)
* https://ki4070ma.github.io/python-client-sphinx/index.html
* After 1.0.0 release, published to https://appium.github.io/python-client/index.html ?
   * Or https://readthedocs.org/

### Reference
* https://www.selenium.dev/selenium/docs/api/py/api.html

### TODO (in another PR)
* More user friendly
* How to handle apis and variables for internal usage
   * e.g. https://ki4070ma.github.io/python-client-sphinx/webdriver.html#module-webdriver.mobilecommand

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### How to generate doc
+
+```bash
```
Comment (ki4070ma) [inline docs/README.md]: Scriptize

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('../appium'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'python_client_doc'
```
Comment (ki4070ma) [inline docs/conf.py]: Change to `Python client 1.0.0 beta`

File: docs/README.md (hunk 1 above)
Comment (ki4070ma) [reply docs/README.md]: Temporal solution for now.
https://github.com/ki4070ma/python-client-sphinx/blob/master/github_pages_deploy.sh

File: appium/webdriver/common/multi_action.py [hunk 3]
```diff
@@ -44,10 +44,10 @@ def add(self, *touch_actions: 'TouchAction') -> None:
             touch_actions (`TouchAction`): one or more TouchAction objects describing a chain of actions to be performed by one finger
 
         Usage:
-            a1 = TouchAction(driver)
-            a1.press(el1).move_to(el2).release()
-            a2 = TouchAction(driver)
-            a2.press(el2).move_to(el1).release()
+            a1 = TouchAction(driver)\n
```
Comment (KazuCocoa) [inline appium/webdriver/common/multi_action.py]: Or a new line?

```
a1 = TouchAction(driver)

a1.press(el1).move_to(el2).release()

a2 = TouchAction(driver)
...
```

File: docs/conf.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('../appium'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Python client 1.0beta'
```
Comment (KazuCocoa) [inline docs/conf.py]: A space before _beta_?

File: appium/webdriver/common/multi_action.py (hunk 3 above)
Comment (ki4070ma) [reply appium/webdriver/common/multi_action.py]: Yes, exactly.
I don't have strong motivation, so I'll take your idea.
(I'll check any other solutions. (e.g. option, plugin))

**Comment by ki4070ma** [comment]
Will do in this PR.

https://github.com/appium/python-client/pull/508#pullrequestreview-363434441

**Pull request merged**